subprojects/platform-jvm/src/integTest/groovy/org/gradle/jvm/toolchain/ShowToolchainsIntegrationTest.groovy

```diff
@@ -43,4 +42,16 @@ class ShowToolchainsIntegrationTest extends AbstractIntegrationSpec {
         outputContains("javaToolchains - Displays the detected java toolchains.")
     }
 
+    def "toolchains log contains progress info about installation suppliers"() {
+        when:
+        settingsFile << """
+                include 'a'
+            """
+        buildFile << ""
+        then:
+        run("javaToolchains", "--debug")
+
+        outputContains(":javaToolchains")
+        outputContains("Discovering toolchains provided via Common Linux Locations")
```

Comment:
It looks like that `registerJavaInstallationSuppliers` register all toolchain suppliers without any OS filtering, but you are right, this might be changed. I'll replace this to something else.